modules/ui/fields/localized.js

```diff
@@ -521,16 +521,52 @@ export function uiFieldLocalized(field, context) {
         loadCountryCode();
         return localized;
     };
+async function fetchCountryData() {
+    const response = await fetch('https://restcountries.com/v3.1/all');
```

Comment:
Loading a third-party resource on demand raises privacy concerns. Anyways, the list of countries doesn’t change often enough to require fetching a new list every time the user uses this field. In the worst case, we could hard-code the list of valid country codes, but we don’t have to do that either: we already depend on [country-coder](https://github.com/rapideditor/country-coder/); the code you replaced refers to this package in order to get the ISO code of the country that the user is currently looking at. You can use this same package to enumerate all the valid country codes.